src/Infrastructure/Controller/Regulation/DuplicateRegulationController.php

```diff
@@ -67,7 +67,7 @@ public function __invoke(Request $request, string $uuid): Response
                 ]),
                 status: Response::HTTP_SEE_OTHER,
             );
-        } catch (OrganizationAlreadyHasRegulationOrderWithThisIdentifierException) {
+        } catch (ValidationFailedException) {
             $session->getFlashBag()->add('error', $this->translator->trans('regulation.duplicated.identifier_error'));
         }
```

Comment:
On ne comprend plus très bien le lien entre `ValidationFailedException` et l'ajout d'une erreur `identifier_error`

Est-ce que le `violatingMessage` contient ce qu'on veut ?

```suggestion
        } catch (ValidationFailedException $exc) {
            $session->getFlashBag()->add('error', $exc->getViolatingMessage());
        }
```

Reply:
On ne peut pas utiliser `getViolatingMessage()` qui est bizarrement l'objet de la commande mais il faut passer par `$exc->getViolations()[0]->getMessage()`.